test/common/upstream/eds_speed_test.cc

```diff
@@ -47,7 +47,7 @@ class EdsSpeedTest {
 
   void resetCluster(const std::string& yaml_config, Cluster::InitializePhase initialize_phase) {
     local_info_.node_.mutable_locality()->set_zone("us-east-1a");
-    eds_cluster_ = parseClusterFromV2Yaml(yaml_config);
+    eds_cluster_ = parseClusterFromV3Yaml(yaml_config);
```

Comment:
This PR has `parseClusterFromV3Yaml`, so does not seem orthogonal to https://github.com/envoyproxy/envoy/pull/11894. It would be helpful if you can split the PRs along some easily understandable line, or stage them one after the other.

Also, friendly ping on updating title/description, it's hard to figure out what "10843 construct" means.

Reply:
@htuch i am not receiving any conflicts message for this PR. please suggest how to proceed.

Reply:
@htuch seems some env issues in my local setup. i  recreated duplicate PR for this PR. please review it. #12072. 
           we can close this now.